tests/unit/LimitedRestorationPanel.spec.ts

```diff
@@ -0,0 +1,77 @@
+import Vue from 'vue'
+import Vuetify from 'vuetify'
+import { createLocalVue, mount, Wrapper } from '@vue/test-utils'
+import { LimitedRestorationPanel } from '@/components/limited-restoration-panel'
+import VueRouter from 'vue-router'
+
+Vue.use(Vuetify)
+let vuetify = new Vuetify({})
+
+const localVue = createLocalVue()
+localVue.use(VueRouter)
+
+/**
+ * Creates and mounts a blank, un-populated component
+ */
+function createDefaultComponent (
+  currentDate: string = '2023-02-03',
+  expiryDate: string = null
+): Wrapper<LimitedRestorationPanel> {
+  return mount(LimitedRestorationPanel, {
+    propsData: {
+      currentDate,
+      expiryDate
+    },
+    vuetify,
+    localVue
+  })
+}
+
+describe('Initialize RelationshipsPanel component', () => {
+  it('loads the component', () => {
+    const wrapper: Wrapper<LimitedRestorationPanel> = createDefaultComponent()
+    expect(wrapper.findComponent(LimitedRestorationPanel).exists()).toBe(true)
+    wrapper.destroy()
+  })
+
+  it('No expiry field exists (going from full to limited restoration), Months: 24', async () => {
```

Comment:
We try (sometimes not very hard) to make this description a sentence, eg, "It is initialized to 24 months if the expiry field is not provided)".